styleguide/checkstyle.xml

```diff
@@ -145,27 +145,32 @@ module PUBLIC "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
     </module>
     <module name="MemberName">
       <property name="format"
-                value="^m_[a-z][a-z0-9][a-zA-Z0-9]*$" />
+                value="^(m_[a-z]([a-zA-Z0-9]*)|value)$" />
```

Comment:
Any particular reason for the redundant group within the `m_` case?

Reply:
Copy pasta is why haha